DESCRIPTION

```diff
@@ -11,8 +11,8 @@ Maintainer: Bruce Hoff <bruce.hoff@sageabse.org>
 BugReports: https://github.com/Sage-Bionetworks/synapser/issues
 Description: Provides R language bindings for Synapse RESTful web services.
 License: Apache License 2.0
-Imports: PythonEmbedInR
-Depends: PythonEmbedInR(>= 0.2), R(>= 3.3), methods
+Imports: PythonEmbedInR(>= 0.2)
+Depends: R(>= 3.3), methods
```

Comment:
should 'methods' be on the Depends line or the Imports line?

Reply:
I think it should be in Imports

Reply:
Though practically it probably does not matter too much since `methods` is one of the base packages that is usually attached on startup anyway.

Reply:
package "methods" is on `Depends:` line because of https://sagebionetworks.jira.com/browse/SYNR-1148
This is because we use `setGeneric` in `.onload`. When running an R script containing the following code:
```
try(remove.packages('PythonEmbedInR'), silent=T)
try(remove.packages('synapser'), silent=T)
install.packages('synapser', repos=c("https://cran.cnr.berkeley.edu/", Sys.getenv('RAN')))
library('synapser')
```
Rscript does not load `methods` automatically.

Reply:
I think we should call `setGeneric()` with its package prefix, i.e. `methods::setGeneric()`. All functions from other packages that aren’t imported into the synapser namespace need to be called in this way for R CMD check to pass anyway.